pages/web-literacy/MakeLinksFromWebLitSkills.jsx

```diff
@@ -0,0 +1,21 @@
+var React = require('react');
+var weblitcontent = require('./weblitcontent');
+var webLitSkillsContent = weblitcontent.webLitSkills;
+var ReactRouter = require('react-router');
+var Link = ReactRouter.Link;
+
+var MakeLinksFromWebLitSkills = function(webLitSkills) {
+  return webLitSkills.map(function(webLitSkill, index) {
+    return (
+      <span className="comma-separated-links" key={webLitSkill}>
```

Comment:
I initially tried that, but the comma ends up as part of the link, because the pseudo element is inside the `<a>`

This span way, the pseudo element is outside of the `<a>` thus it gets the effect we want with links separate from the comma.